README.md

```diff
@@ -253,6 +253,7 @@ The second snippet shows how to create records inside a zone. The complete code
 
 ```java
 import com.google.gcloud.dns.ChangeRequest;
+import com.google.gcloud.dns.ChangeRequestInfo;
```

Comment:
I believe the `ChangeRequest` import is no longer needed. Might be wrong though.